### PR TITLE
Remove deprecated consensys ERC-20 implementation

### PR DESCRIPTION
**Pull request by MatthieuScarset**
## Description

The [repo by ConsenSys(https://github.com/ConsenSys/Tokens#deprecated) is marked as deprecated.

I think it's safer to tell readers about it before redirecting them to the EIP20 contract page directly. One might not necessarily think about verifying the main README file and would reuse/copy-paste EIP20.

### CODE REVIEW

**Comment by minimalsm** [comment]
Hey @MatthieuScarset, thanks for the heads up on this 🤘

Curious for your input here: wouldn't it be better to remove it seeing as we have other, up to date and maintained implementations of ERC-20 already listed?

**Comment by MatthieuScarset** [comment]
Yes I would understand if we prefer to remove the link completely 🚮

I simply marked it "deprecated" as it was worth it for me to go read the code and compare both contracts.

**Comment by corwintines** [comment]
I definitely +1 to deprecating it and just removing the link in favor of the OpenZepplin contracts that are linked out to.

File: src/content/developers/docs/standards/tokens/erc-20/index.md [hunk 1]
```diff
@@ -153,4 +153,4 @@ print("Addr Balance:", addr_balance)
 - [EIP-20: ERC-20 Token Standard](https://eips.ethereum.org/EIPS/eip-20)
 - [OpenZeppelin - Tokens](https://docs.openzeppelin.com/contracts/3.x/tokens#ERC20)
 - [OpenZeppelin - ERC-20 Implementation](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/ERC20.sol)
-- [ConsenSys - ERC-20 Implementation](https://github.com/ConsenSys/Tokens/blob/master/contracts/eip20/EIP20.sol)
+- [ConsenSys - ERC-20 Implementation](https://github.com/ConsenSys/Tokens/blob/master/contracts/eip20/EIP20.sol) (deprecated)
```
Comment (minimalsm) [inline src/content/developers/docs/standards/tokens/erc-20/index.md:156]: I'm definitely in favour of just removing it.

1. The page is primarily solving for users who want to know what ERC-20 is or want to know how to implement it. I don't think keeping this in solves for either user group.
2. Defaulting to remove deprecated/unmaintained software makes it easy to make the change across all translated languages on the site  (the alternative being that we queue up translations for adding a deprecated label across ~40 languages every time something is out of date)

```suggestion
```

**Review by minimalsm — CHANGES_REQUESTED** [review]

**Comment by minimalsm** [comment]
Cool :-) To get this merged quick but keep @MatthieuScarset as last edited author I'm going to merge this and throw up another PR to remove the deprecated software section.

**Comment by minimalsm** [comment]
@MatthieuScarset thanks again for the PR ❤️🚀

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And remember to [claim your POAP](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by minimalsm** [comment]
@all-contributors please add @MatthieuScarset for docs

**Pull request merged**